lib/linkedin/linkedin.profileViews.service.ts

```diff
@@ -0,0 +1,41 @@
+import { CDPSession, Page } from "puppeteer";
+
+import { timer } from "../helpers/timer";
+
+import { LinkedinAbstractService } from "./linkedin.abstract.service";
+import { LinkedinServicesInterface } from "./linkedin.services.interface";
+
+export class LinkedinProfileViewsService
+  extends LinkedinAbstractService
+  implements LinkedinServicesInterface<{}>
+{
+  globalError() {
+    return {
+      text: "Could not scrape pages",
+    };
+  }
+  async process(page: Page, cdp: CDPSession) {
+    await page.goto("https://www.linkedin.com/analytics/profile-views/");
+    await timer(3000);
```

Comment:
You need to wait for an element, 3000 is not good enough because some people have slow internet

Reply:
Yeah fixed in the recent commit